pkg/operator/ceph/cluster/osd/topology/topology.go

```diff
@@ -56,66 +57,69 @@ func ExtractOSDTopologyFromLabels(labels map[string]string) (map[string]string,
 	return topology, topologyAffinity
 }
 
+func rookTopologyLabelsOrdered() []string {
+	topologyLabelsOrdered := []string{}
+	for i := len(CRUSHTopologyLabels) - 1; i >= 0; i-- {
+		label := CRUSHTopologyLabels[i]
+		topologyLabelsOrdered = append(topologyLabelsOrdered, topologyLabelPrefix+label)
+	}
+	return topologyLabelsOrdered
+}
+
+func allKubernetesTopologyLabelsOrdered() []string {
+	return append([]string{corev1.LabelFailureDomainBetaRegion, // deprecated label for backwards compatibility
```

Comment:
We would just remove it in the master as per the discussion 
PR: https://github.com/rook/rook/pull/11836

Reply:
It's such a small change it might be easier as just a separate commit in this PR, but either way is fine.

Reply:
With that comment I was assuming it would be an independent change even if in the same file, but since you've done some refactoring, it makes sense now to do it at the same time.